soh/soh/GameMenuBar.cpp

```diff
@@ -302,6 +302,17 @@ namespace GameMenuBar {
                 UIWidgets::EnhancementRadioButton("French", "gLanguages", LANGUAGE_FRA);
                 ImGui::EndMenu();
             }
+            
+#if defined(_WIN32) || defined(__APPLE__)
```

Comment:
is there a reason this is windows/mac only? i'd think instead of having the whole menu hidden it'd make sense to hide individual options that are platform specific